src/test/java/com/kenai/jffi/VersionTest.java

```diff
@@ -0,0 +1,44 @@
+package com.kenai.jffi;
+
+import org.junit.Test;
+
+import com.kenai.jffi.InvokerTest.HeapInvoker;
+import com.kenai.jffi.InvokerTest.NativeInvoker;
+import com.kenai.jffi.UnitHelper.Address;
```

Comment:
Is it OK to import this stuff from other tests?

Reply:
Yeah that should be fine. We're not too strict about how tests are structured.